bazel/grpc_deps.bzl

```diff
@@ -2,6 +2,12 @@
 
 def grpc_deps():
     """Loads dependencies need to compile and test the grpc library."""
+
+    native.bind(
```

Comment:
@nicolasnoble I've kept it as `bind` for consistency with the rest. I believe further changes would be needed (in particular, in the `_get_external_deps` function) in order to use `local_repository`, meaning we ought to make the `bind` -> `local_repository` atomically.

Reply:
For Asylo, we would need to bind our own nanopb to defined PB_FIELD_16BIT, but we also call grpc_deps() after we load our own dependencies. I don't know if bind of the same name is legal (I imagine it isn't), so this probably won't work for us as a solution.